cmd/kube-scheduler/app/server.go

```diff
@@ -127,6 +127,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.config.SchedulerName, "scheduler-name", o.config.SchedulerName, "Name of the scheduler, used to select which pods will be processed by this scheduler, based on pod's \"spec.SchedulerName\".")
 	fs.StringVar(&o.config.LeaderElection.LockObjectNamespace, "lock-object-namespace", o.config.LeaderElection.LockObjectNamespace, "Define the namespace of the lock object.")
 	fs.StringVar(&o.config.LeaderElection.LockObjectName, "lock-object-name", o.config.LeaderElection.LockObjectName, "Define the name of the lock object.")
+	fs.Int32Var(&o.config.ResyncIntervalInSec, "resync-interval", o.config.ResyncIntervalInSec, "The interval in seconds local cache is rebuilt and compared against existing cache.")
```

Comment:
Is this prob only for debug purpose? In that case we need to clarify it.

Reply:
done.